phalcon/cache/backend/memcache.zep

```diff
@@ -117,6 +117,28 @@ class Memcache extends Backend implements BackendInterface
 		let this->_memcache = memcache;
 	}
 
+        /**
+	 * Add servers to memcache pool
+	 *
+	 * @param  string host
+	 * @param  long port
+         * @param  boolean persistent	 
+         * @return boolean
+	 */
+	public function addServers(var host, var port, var persistent = false)
```

Comment:
I think should use such declaration:
```zephir
public function addServers(string! host, int port, boolean persistent = false) -> boolean
```

and remove `@param`s doc

See please PHP.net doc:

```php
bool Memcache::addServer ( string $host [, int $port = 11211 [, bool $persistent [, int $weight [, int $timeout [, int $retry_interval [, bool $status [, callable $failure_callback [, int $timeoutms ]]]]]]]] )
```

What with other params?